src/projects/diff/hunk-sorter.html

```diff
@@ -159,6 +170,11 @@
         if (unsortedGroup.diff.length > 0) {
           allGroups.push(unsortedGroup);
         }
+        if (fbOrdering.groups === undefined && this.backendPath && defaultGroup.diff.length > 0) {
+          this.$.checkIfPathIsThere.body = defaultGroup;
```

Comment:
Maybe we should alter the name of the default group to show the user that the changes were directly imported from GitHub